Jenkinsfile

```diff
@@ -632,8 +648,71 @@ pipeline {
       }
     }
 
+    stage('Deploy Preview') {
+      /*
+      when {
+        not {
+          branch 'PR-*'
+        }
+      }
+      */
+      steps {
+        setGitHubBuildStatus('nuxeo/preview', 'Deploy nuxeo preview', 'PENDING')
+        container('maven') {
+          dir('ci/helm/preview') {
+            echo """
+            ----------------------------------------
+            Deploy Preview environment
+            ----------------------------------------"""
+            // first substitute environment variables in chart values
+            sh """
+              mv values.yaml values.yaml.tosubst
+              envsubst < values.yaml.tosubst > values.yaml
+            """
+            // second create target namespace (if doesn't exist) and copy secrets to target namespace
+            script {
+              boolean nsExists = sh(returnStatus: true, script: "kubectl get namespace ${PREVIEW_NAMESPACE}") == 0
+              String noCommentOpt = '';
+              if (nsExists) {
+                noCommentOpt = '--no-comment'
+                // Previous preview deployment needs to be scaled to 0 to be replaced correctly
+                sh "kubectl -n ${PREVIEW_NAMESPACE} scale deployment nuxeo-preview --replicas=0"
+              } else {
+                sh "kubectl create namespace ${PREVIEW_NAMESPACE}"
```

Comment:
I believe that `jx step helm install` creates the namespace if it doesn't exist, so this might be useles..

Reply:
indeed, so it did with my minikube